prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -67,12 +67,12 @@ class ReportFunction {
         ) request: HttpRequestMessage<String?>,
         context: ExecutionContext,
     ): HttpResponseMessage {
-        try {
```

Comment:
These changes are part of fixing ReportFunction to return more cleanly on Exceptions